src/plugins/vis_types/timeseries/public/application/visualizations/views/_gauge.scss

```diff
@@ -3,6 +3,12 @@
  *    are being calcuated from a percentage of the base font size of 100% (14px).
  */
 
+$tvbTextColor: transparentize($euiColorFullShade, .4);
```

Comment:
You repeated this block a bunch of times, would it make sense to keep it in a separate field and import it in these places instead?

Reply:
Hm, make sense. I am just tried avoid creating another file with styles.

Reply:
done